lib/round.rb

```diff
@@ -0,0 +1,78 @@
+require 'pry'
+require_relative'./deck'
+require_relative'./turn'
+require_relative './card'
+
+class Round
+    attr_reader :deck, :turns, :correct_answer
```

Comment:
There's an `attr_reader` here for `:correct_answer` but I don't see where it is ever being used.